packages/ui/routes/mailings/__partials/mailings-filters.vue

```diff
@@ -1,5 +1,8 @@
 <script>
-import * as apiRoutes from '~/helpers/api-routes.js';
+import { FOLDER, SET_FILTERS } from '~/store/folder';
+import { TEMPLATE } from '~/store/template';
+import { mapState } from 'vuex';
+import _ from 'lodash';
```

Comment:
```suggestion
import debounce from 'lodash/debounce';
```

Never never import lodash as a whole; only import the needed function :)

Reply:
Roger that :+1: